org.spesmilo.Electrum.json

```diff
@@ -0,0 +1,103 @@
+{
+  "app-id": "org.spesmilo.Electrum",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "command": "electrum",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=host",
+    "--filesystem=xdg-config/kdeglobals:ro"
+  ],
+  "modules": [
+    {
+      "name": "pyqt5",
+      "config-opts": [
+        "--disable-static",
+        "--enable-x11"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.11.2/PyQt5_gpl-5.11.2.zip",
+          "sha256": "4a83de09fb8e5b094c14a5900c6f1f117c00798a24eaf09377cf469ddd016009"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "python3 configure.py --assume-shared --confirm-license --no-designer-plugin --no-qml-plugin --sysroot=/app --verbose --qsci-api --qsci-api-destdir=/app/qsci --sipdir=/app/share/sip --sip=/app/bin/sip --sip-incdir=/app/include QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.7m/' QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.7m/'"
+          ],
+          "dest-filename": "configure"
+        }
+      ],
+      "modules": [
+        {
+          "name": "sip",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.13/sip-4.19.13.tar.gz",
+              "sha256": "e353a7056599bf5fbd5d3ff9842a6ab2ea3cf4e0304a0f925ec5862907c0d15e"
+            },
+            {
+              "type": "script",
+              "commands": [
+                "python3 configure.py --sip-module PyQt5.sip -b ${FLATPAK_DEST}/bin -d ${FLATPAK_DEST}/lib/python3.7/site-packages -e ${FLATPAK_DEST}/include -v ${FLATPAK_DEST}/share/sip --stubsdir=${FLATPAK_DEST}/lib/python3.7/site-packages"
+              ],
+              "dest-filename": "configure"
+            }
+          ],
+          "cleanup": [
+            "/bin",
+            "/include"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "electrum",
+      "buildsystem": "simple",
+      "build-commands": [
+        "tar -xzf Electrum-3.3.2.tar.gz",
```

Comment:
You could use the *archive source* for that.

http://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#idm45850111393920